plugins/SliceInfoPlugin/example_data.html

```diff
@@ -0,0 +1,40 @@
+Cura Version: 4.0<br/>
```

Comment:
There is one set of data that is not in the example, we also send some "per object" data (eg; transformation meshes, mode hashes and axis aligned bounding boxes)

Reply:
Added them and improved the HTML file in c26dbfd